cmd/dcrdata/public/scss/_variables.scss

```diff
@@ -52,3 +52,13 @@ $card-bg-secondary: #f8f8f9;
 $card-bg-secondary-dark: #222326;
 
 $dark-link-hover-color: #94ffca;
+
+// links
+$link-decoration: none;
+$link-hover-decoration: underline;
+
+// variables that changed with the bootstrap 5 migration.
+$input-bg: #fff;
+$input-color: #495057;
+$nav-link-color: inherit;
+// $min-contrast-ratio: 3.9; // contrasting color for the primary (Decred blue) depends on contrast factor
```

Comment:
For posterity, we talked a lot about the Decode button's text color and it came down to the change in Bootstrap's color contast system. 
https://getbootstrap.com/docs/5.0/migration/#sass

> Renamed color-yiq() function and related variables to color-contrast() as it’s no longer related to YIQ colorspace

What that doesn't explain is that the new WCAG algo potentially changes the contrasting color that gets picked, in this case for Decred blue.  

https://github.com/twbs/bootstrap/blob/57d80fcd3274ba1ff0a1580966e098fda80f74e7/scss/_variables.scss#L97
https://getbootstrap.com/docs/5.0/customize/sass/#color-contrast
https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast

In the end, I concluded it was pretty much an accident that this button was using white as the text color, and it really should have the same text as the other (Broadcast) button and with an high contrast in terms of WCAG.